2017-2018/GameEngine/graphics/variables.py

```diff
@@ -1,6 +1,6 @@
 from enum import Enum
 
-DISPLAY_FREQUENCY = 1/60 # 60 Hz
+DISPLAY_FREQUENCY = 1./240 # 60 Hz
```

Comment:
In Python 3, the decimal point is no longer needed, it will automatically convert integers into floats on division if necessary.

Reply:
I'll remove it